### PR TITLE
chore: Add instructions on debugging on CircleCI

### PR DESCRIPTION
**Pull request by dylanb**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -136,3 +136,43 @@ describe('Module', () => {
...
+
+```
+ssh -X -p 64605 ubuntu@13.58.157.61
+```
+
+When you login, set up the environment and start the chrome browser
+
+```
+export DISPLAY=localhost:10.0
+/opt/google/chrome/chrome
```
Comment (marcysutton) [inline CONTRIBUTING.md]: Are these separate lines? I get this result:

```
ubuntu@box1325:~$ export DISPLAY=localhost:10.0
ubuntu@box1325:~$ /opt/google/chrome/chrome
[23929:23929:0624/220821:ERROR:browser_main_loop.cc(260)] Gtk: cannot open display: localhost:10.0
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -136,3 +136,43 @@ describe('Module', () => {
...
+```
+
+### .Xauthority does not exist
+
+Edit the ~/.Xauthority file and just save it with the following commands
```
Comment (marcysutton) [inline CONTRIBUTING.md]: What commands? It looks like you open the file and then close it without writing anything?

Comment (dylanb) [inline CONTRIBUTING.md]: yep

File: CONTRIBUTING.md (hunk 1 above)
Comment (dylanb) [inline CONTRIBUTING.md]: Did you get the warning about .Xauthority?

**Comment by marcysutton** [comment]
The Chrome browser that opens is extremely slow, but it does connect to the tests. The only part I couldn't get to work was the `Xauthority` thing.

File: CONTRIBUTING.md (hunk 1 above)
Comment (marcysutton) [inline CONTRIBUTING.md]: I did, in both login versions (with and without -X)

**Comment by dylanb** [comment]
@marcysutton were you ot able to open and close the ~/.Xauthority file?

**Comment by marcysutton** [comment]
@dylanb I opened and closed it, but I still got the warning after that.

**Comment by dylanb** [comment]
@marcysutton as long as it works, the warning can be ignored.

**Comment by marcysutton** [comment]
Roger! I was able to get the tests to run, it was just extremely slow. Not sure what can be done about that.

**Comment by dylanb** [comment]
@marcysutton nothing can be done, you are running X-Windows across the Internet - it is bound to be slow

**Pull request merged**